application/views/userManagement/index.php

```diff
@@ -21,7 +21,6 @@
 
 <?php $this->renderPartial('partial/_menubar'); ?>
 
-<div class="row">
     <?php
```

Comment:
Do we need such comments like "Example: Datepicker etc..." inside the codebase?

Reply:
no we don't, but not really part of the task to remove it. Probably would have removed it if i saw it :)